repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/featured-content.ftl

```diff
@@ -3,7 +3,7 @@
 <#include "../utils/date-util.ftl">
 <#include "../macros/internal-link.ftl">
 
-<#macro featuredContent block maxCards=2>
+<#macro featuredContent block maxCards=3>
```

Comment:
I think the `maxCards=3` change will affect pages wherein the featured content block is rendered as a content block.

If we keep `maxCards=2`, then look to pass the `maxCards=3` argument when called from page furniture-based templates (e.g. Home page, Publication landing page, etc.)

If we keep `maxCards=3`, then look to pass the `maxCards=2` argument when called from content block-based templates (e.g. Blog post, SCP, etc.)